examples/lua/README.md

```diff
@@ -0,0 +1,64 @@
+
```

Comment:
please rename this file to nats_adapter.md
in the future we would have other examples here as well